docs/content/operations/metrics.md

```diff
@@ -87,6 +87,7 @@ Memcached client metrics are reported as per the following. These metrics come d
 |`ingest/events/thrownAway`|Number of events rejected because they are outside the windowPeriod.|dataSource.|0|
 |`ingest/events/unparseable`|Number of events rejected because the events are unparseable.|dataSource.|0|
 |`ingest/events/processed`|Number of events successfully processed.|dataSource.|Equal to your # of events.|
+|`ingest/events/buffered`|Number of events queued in the EventReceiverFirehose's buffer|serviceName, bufferCapacity.|Equal to current # of events in the buffer queue.|
```

Comment:
I believe this metric will not get published by default, user would have to add that to druid.monitor.monitors property ?